awx_collection/plugins/lookup/controller_api.py

```diff
@@ -11,8 +11,8 @@
 requirements:
   - None
 description:
-  - Returns GET requests from the Automation Platform Controller API. See
-    U(https://docs.ansible.com/ansible-tower/latest/html/towerapi/index.html) for API usage.
+  - Returns GET requests from the AWX API. See the AWX API documentation
+    U(https://ansible.readthedocs.io/projects/awx/en/latest/rest_api/index.html) for API usage.
   - For use that is cross-compatible between the awx.awx and ansible.controller collection
```

Comment:
Then it doesn't distinguish the community vs. the paid version. That is where we get into trouble. I have a discussion point with gundalow about this so we can sort out this very thing.

Reply:
We seem to only include docs links in a few places, can we use a variable there? Seems like we already do some substitution as part of the collection build process

Reply:
After some discussion with @gundalow, I'll point readers to a [landing page](https://docs.ansible.com/automation-controller/latest/html/towerapi/) that has links for them to go to, depending on their situation. Since this page will ultimately go away, we will keep this an open issue (tracked elsewhere) until we are directed to do something different.